docker/entrypoint.sh

```diff
@@ -19,23 +35,28 @@ else
   ipfs config Addresses.Gateway /ip4/0.0.0.0/tcp/8080
 fi
 
+echo "Confirming experimental container"
 ipfs daemon --migrate=true &
 sleep 3
 
 # Test whether the mounted directory is writable for us
-if [ ! -w "$IPFS_CLUSTER_PATH" 2>/dev/null ]; then
-  echo "error: $IPFS_CLUSTER_PATH is not writable for user $user (uid=$(id -u $user))"
+if [ ! -w "$cluster_repo" 2>/dev/null ]; then
+  echo "error: $cluster_repo is not writable for user $user (uid=$(id -u $user))"
   exit 1
 fi
 
 ipfs-cluster-service --version
 
-if [ -e "$IPFS_CLUSTER_PATH/service.json" ]; then
+if [ -e "$cluster_repo/service.json" ]; then
     echo "Found IPFS cluster configuration at $IPFS_CLUSTER_PATH"
 else
     ipfs-cluster-service init
     sed -i 's/127\.0\.0\.1\/tcp\/9094/0.0.0.0\/tcp\/9094/' "$IPFS_CLUSTER_PATH/service.json"
     sed -i 's/127\.0\.0\.1\/tcp\/9095/0.0.0.0\/tcp\/9095/' "$IPFS_CLUSTER_PATH/service.json"
 fi
 
-exec ipfs-cluster-service $@
+ipfs-cluster-service $@ &
```

Comment:
From the description of `tini`, it sounded like it may solve the problems you had when sending signals to the ipfs-cluster-service without the need  to launch in the background. Once we merge the other PR, you should test that signaling the process still doesn't work.